sdk-setup/src/mb2

```diff
@@ -2573,6 +2575,9 @@ run_build__process_args() {
             --no-check) shift
                 BUILD_NOCHECK_ARGS=(--nocheck)
                 ;;
+            --no-rpmlint) shift
+                BUILD_NORPMLINT=1
```

Comment:
Can we go without the option and simply decide with `BUILD_NORPMLINT=${SAILFISH_SDK_FRONTENT:+1}`? There should be little need to use `--no-rpmlint` on user side and it would also remove the need to adapt on sfdk side.

Reply:
As this topic was discussed in sw sync today, I felt that the general consensus was that indeed most users would want to have the option to disable rpmlint.

Reply:
This feels like turning all this effort useless :)

Anyway I would offer an alternative suggestion then: derive the default value as I suggested above `BUILD_NORPMLINT=${SAILFISH_SDK_FRONTENT:+1}` and allow to use the existing `--no-check` option to suppress it under SDK chroot. I don't see strong need for disabling rpmlint if its failure is not fatal, so disabling all checks in the occasional cases should be OK.

Reply:
> This feels like turning all this effort useless :)

Maybe. Time will tell.

> Anyway I would offer an alternative suggestion then: derive the default value as I suggested above `BUILD_NORPMLINT=${SAILFISH_SDK_FRONTENT:+1}` and allow to use the existing `--no-check` option to suppress it under SDK chroot. 

Now that would be clever! https://github.com/sailfishos/sailfish-qtcreator/pull/522 could then be discarded, right?

Reply:
yes

Reply:
The initialization is good now, just the option handling and docs is still left there.

Reply:
Yes, the option handling and docs were left there on purpose.

Reply:
My most recent suggestion was to "... and allow to use the existing --no-check option to suppress it" and you said that would be clever :)

Reply:
I should learn how to read

Reply:
And yes, it is/was even more clever than I thought :)

Reply:
better now?

Reply:
great! :)